client/src/App.js

```diff
@@ -3,27 +3,31 @@ import FloatingActionButton from './components/FloatingActionButton'
 import SearchBooks from './components/SearchBooks'
 import styled from 'styled-components/macro'
 import useBookData from './hooks/useBookData'
+import BookDetail from './components/BookDetail'
 
 export default function App() {
   const {
     addBook,
+    book,
+    bookDetailModal,
     deleteBook,
     handleChange,
     handleSubmit,
     results,
     searchBooksModal,
     selectedBooks,
     setResult,
+    showDetail,
     toggleSearchBooksModal,
   } = useBookData()
```

Comment:
I'm not exactly sure how you chose the order here, but maybe I would adjust it a little thematically.